include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -788,7 +818,7 @@ class MGTwoLevelTransferNonNested<dim,
    * Object to evaluate shape functions on one mesh on visited support points of
    * the other mesh.
    */
-  Utilities::MPI::RemotePointEvaluation<dim> rpe;
+  std::shared_ptr<Utilities::MPI::RemotePointEvaluation<dim>> rpe;
```

Comment:
The change to shared_ptr is not needed!?

Reply:
Sure, it was a leftover from debugging. Fixed, thanks